packages/dashboard/src/hooks/useTermsAgreed.ts

```diff
@@ -0,0 +1,16 @@
+import { useCallback, useState } from 'react'
+import { createContainer } from 'unstated-next'
+
+const KEY = 'dashboard/terms-agreed'
```

Comment:
terms should have a version. e.g. `term-2023-01`, otherwise we don't know if the user accepts the latest term or not